e3sm_diags/driver/qbo_driver.py

```diff
@@ -412,10 +413,10 @@ def _get_psd_from_wavelet(data: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
     """
     deg = 6
     period = np.arange(1, 55 + 1)
-    freq = 1 / period
+    widths = deg / (2 * np.pi / period)
 
-    widths = deg / (2 * np.pi * freq)
-    cwtmatr = scipy.signal.cwt(data, scipy.signal.morlet2, widths=widths, w=deg)
-    psd = np.mean(np.square(np.abs(cwtmatr)), axis=1)
+    [cfs, freq] = pywt.cwt(data, scales=widths, wavelet="cmor1.5-1.0")
+    psd = np.mean(np.square(np.abs(cfs)), axis=1)
+    period = 1 / freq
```

Comment:
Code changes are based on this [comment](https://github.com/E3SM-Project/e3sm_diags/issues/912#issuecomment-2593760489)